oeps/processes/oep-0062-proc-working-groups.rst

```diff
@@ -0,0 +1,191 @@
+
+Even before this OEP was codified, working groups played a vital role in sustaining and advancing the Open edX platform. In this OEP we aim to document their successful practices for the benefit of current and future Open edX community members.
+
+Background & Motivation
+***********************
+
+The first community-lead Open edX working groups were the *Transifex Working Group* and the *Build-Test-Release Working Group*. The former group took responsibility for creating and curating translations of strings on the Open edX platform; the latter took responsility for the platform's named releases. These groups were formed by community members who saw a need in the project and took the initiative to organize and solve that need.
+
+Since then, several more working groups have organized themselves using a similar model. The groups operate autonomously, but have largely standardized around a set of best practices to foster an inclusive and productive environment. Many of the groups play important roles in the project, such as organizing community events, developing standard user interface components, and driving project strategy. As a whole, working groups provide a space for Open edX contributors from different organizations and geographic regions to collaborate and form working relationships.
+
+Given the importance of working groups to the health of the Open edX project, it may surprise new Open edX contributors that litle formal documentation exists to establish the working group system. The only official guidance so far is found in the `Technical Oversight Committee (TOC) Charter`_:
+
+    **10. Working Groups**
+
+    a. From time-to-time as needed the TOC may create (and wind down) working groups or task forces that will be responsible for recommending and delivering critical project work and reporting on progress and results to the TOC.
+
+    b. The TOC will typically delegate tactical decision making to those who best understand the details, including Working Groups, Task Forces, Expert Groups, etc. Only significant changes in the status quo or decisions that are difficult to reverse should be escalated to the TOC for consideration
+
+While this excert is true, TOC-created working groups are only a special case of the broader working group system. In this document, we aim to capture the generally-applicable processes and best practices for working groups so that they can be more readily understood by newcomers to the Open edX project and more easily referenced by all community members. In order to avoid disruptive the already-successful working group system that exists today, this OEP mandates only a lightweight framework, one which mostly aligns with how the community already operates.
```

Comment:
```suggestion
While this excerpt is true, TOC-created working groups are only a special case of the broader working group system. In this document, we aim to capture the generally-applicable processes and best practices for working groups so that they can be more readily understood by newcomers to the Open edX project and more easily referenced by all community members. In order to avoid disrupting the already-successful working group system that exists today, this OEP mandates only a lightweight framework, one which mostly aligns with how the community already operates.
```